noir-projects/aztec-nr/address-note/src/address_note.nr

```diff
@@ -12,15 +12,16 @@ global ADDRESS_NOTE_LEN: Field = 3;
 #[aztec(note)]
 struct AddressNote {
     address: AztecAddress,
-    owner: AztecAddress,
+    // The nullifying public key hash of the person who owns the note. 
+    // This is used with the app_nullifier_secret_key to ensure that the note can be privately spent.
+    owner_npk_m_h: Field,
```

Comment:
```suggestion
    npk_m_hash: Field,
```
I think it's pretty clear that the npk_m here is relevant for note nullification so I don't think having owner there is necessary. Also the owner might be confusing in escrows.

"h" there was a bit unclear so I think just having "hash" there would be better.